code/game/dna/genes/vg_powers.dm

```diff
@@ -215,23 +215,37 @@
 	var/list/targets = new /list()
 	var/list/validtargets = new /list()
 	var/turf/T = get_turf(user)
+	var/list/mobs_in_view = user.get_visible_mobs()
+	var/list/names = list()
+	var/unknowns = 1
+	
 	for(var/mob/living/M in range(14, T))
 		if(M && M.mind)
 			if(M == user)
 				continue
-			validtargets += M
+			var/mob_name
+			if(M in mobs_in_view)
+				mob_name = M.name
+			else
+				mob_name = "Unknown entity([unknowns++])" // Avoid dupes
+			if(validtargets[name])
+				mob_name = "[mob_name](2)"
+			names += mob_name
+			validtargets[mob_name] = M
 
 	if(!validtargets.len)
 		to_chat(user, "<span class='warning'>There are no valid targets!</span>")
 		start_recharge()
 		return
 
-	targets += input("Choose the target to talk to.", "Targeting") as null|mob in validtargets
+	var/target_name = input("Choose the target to listen to.", "Targeting") as null|anything in names
```

Comment:
I believe that's supposed to be 'Choose the target to talk to'

Reply:
Good catch. Yep copy pasta fail